### ISSUE TITLE
Could you add different styles of border and axes in 3D graphics? 

### ISSUE THREAD

**Issue by Photonico** [sciapp/gr#86]
GR is an excellent visualization solution. But I have a little dissatisfaction with the style of border and axes plotting. GR is almost perfect but I dislike its sloping tics.
I prefer the style of Gnuplot, MathGL, and PGFPlot. 

Could you develop it?

Gnuplot Sample:
![Gnuplot Sample](http://gnuplot.sourceforge.net/demo_5.3/fenceplot.1.png)

MathGL Sample:
![MathGL Sample](http://mathgl.sourceforge.net/png/cont3.png)

PGFPlot Sample:
![PGFPlot Sample](https://kristofferc.github.io/PGFPlotsX.jl/stable/examples/mesh-scatter.svg)

Sincerely,

**Comment by Photonico** [comment]
Hello,
Could you please consider it?
Sincerely,

**Comment by jheinen** [comment]
Do you mean the "slanted" text labels or the tics?

**Comment by Photonico** [comment]
> 
> 
> Do you mean the "slanted" text labels or the tics?

Yes. Could you add a statement which can control whether to display "slanted" text or "normal" text? I really like GR in Python and Julia, but I don't like the "slanted" text.
About tics, I think you can also make different styles. It will be great.

Sincerely,

**Comment by Photonico** [comment]
> 
> 
> Do you mean the "slanted" text labels or the tics?

Will you implement this function?

**Comment by jheinen** [comment]
I could add an attribute how labels for 3d axes are displayed ...

**Comment by Photonico** [comment]
> 
> 
> I could add an attribute how labels for 3d axes are displayed ...

Great! This change will be thankful! When will you solve it?
I want to use GR.jl in my current article.
